src/Core/Components/InfoButton/FluentInfoButton.razor

```diff
@@ -0,0 +1,20 @@
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
+@inherits FluentComponentBase
+
+<FluentIcon Id="@Id"
+            Value="@(_showIconFilled ? IconFilled : IconOutline)"
+            Width="@IconWidth"
+            Color="IconColor"
+            Focusable="true"            
+            CustomColor="@IconCustomColor"
+            @onmouseout="@OnIconMouseOut"
+            @onmouseover="@(() => _showIconFilled = true)"
+            @onfocusout="@OnIconFocusOut"
+            OnClick="@(() => _visible = !_visible)" />
```

Comment:
Can you use `@OnClick` method (same for `onmouseover`) to centralize the C# code in the .cs file?

Reply:
I don't understand they are two different things

Reply:
Not to use `OnClick="@(() => _visible = !_visible)" />` but `OnClick="@OnClickHander" />`. And the code in the .cs file.

Reply:
It's more Clean Code: it's all about having the code in the same file,

Reply:
you are masters. "your kung fu is very strong" 

![image](https://github.com/microsoft/fluentui-blazor/assets/6276309/f56a42ce-2bbc-48bb-adb1-08dc3364758c)